sdk/maps/maps-route-rest/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@azure/azure-sdk/ts-modules-only-named": "warn",
+    "@azure/azure-sdk/ts-apiextractor-json-types": "warn",
+    "@azure/azure-sdk/ts-package-json-types": "warn",
+    "@azure/azure-sdk/ts-package-json-engine-is-present": "warn",
+    "tsdoc/syntax": "warn"
```

Comment:
Can you check if you can remove any of these and have `rushx lint` succeed?

Reply:
"@azure/azure-sdk/ts-package-json-engine-is-present" will fail, but I just update `engine.node` from `>=12.0.0` to `>=14.0.01` and it works fine